tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -144,7 +144,11 @@ initialize_column_state(DecompressChunkState *state)
 		next_compressed_scan_attno++;
 
 		AttrNumber output_attno = lfirst_int(lc);
-		Assert(output_attno != 0);
+		if (output_attno == 0)
+		{
+			/* skip this column from compressed scan targetlist, do nothing */
+			continue;
+		}
```

Comment:
Add a comment on why this is OK.

Reply:
That's the intended behavior, output_attno == 0 means 'skip this column'. I reworded the comment.